scripts/dts/extract_dts_includes.py

```diff
@@ -641,7 +641,12 @@ def generate_include_file(defs, args):
         sys.stdout.write('/* ' + node.split('/')[-1] + ' */')
         sys.stdout.write("\n")
 
-        maxlength = max(len(s + '#define ') for s in defs[node].keys())
+        max_dict_key = lambda d: max(len(k) for k in d.keys())
```

Comment:
You can simplify this:

```Python
longest_key = max(d, key=len)
longest_alias = max(defs[node].get('aliases', {}), key=len)
maxlength = len(max(longes_key, longest_alias)) + len('#define ')
```

Reply:
@pfalcon, I didn't notice the typo until you explain it again. Thanks.
@lpereira, max() doesn't allow empty sequence argument and max() will return the string with max alphabetical character.